### DISCUSSION TITLE
Support for rich==13*

### DISCUSSION THREAD

**Discussion by mgorny** [encode/httpx#2544] [Potential Issue] [answered]
FWICS all tests pass after upgrading `rich` to 13.0.0. Could you please consider raising the pin to `<14`?

**Comment by lovelydinosaur** [comment, marked as answer]
Seems fine to me. Would you like to open a pull request bumping the version requirements?

**Reply by mgorny** [reply]
https://github.com/encode/httpx/pull/2546
